jaseci_core/jaseci/jsctl/jac_talk_utils.py

```diff
@@ -0,0 +1,130 @@
+import os
+import pathlib
+import shutil
+import tempfile
+
+import git
+from jaseci.actions.live_actions import load_module_actions
+from jaseci.svc import MetaService
+from jaseci.utils.utils import logger
+
+curr_path = pathlib.Path().resolve()
+
+
+class JacExecutable:
+    def __init__(self, run_svcs=False):
+        self.meta = MetaService(run_svcs=run_svcs)
+        self.smast = self.meta.build_super_master()
+        self.mast = self.meta.build_master(h=self.smast._h)
+
+    def call(self, mast, pl):
+        ret = mast.general_interface_to_api(api_name=pl[0], params=pl[1])
+        return ret
+
+    def load_jac(self, fn):
+        with open(fn) as f:
+            return f.read()
+
+    def train(self, jac_file, train_file):
+        self.call(
+            self.mast,
+            ["sentinel_register", {"code": self.load_jac(jac_file)}],
+        )
+        return self.call(
+            self.mast,
+            ["walker_run", {"name": "train", "ctx": {"train_file": train_file}}],
+        )
+
+    def run_main_jac(self, jac_file):
+        self.call(
+            self.mast,
+            ["sentinel_register", {"code": self.load_jac(jac_file)}],
+        )
+        return self.call(self.mast, ["walker_run", {"name": "init"}])
+
+
+def update_file(infile):
+    data = open(infile, "r")
+    find = 'import {*} with "./graph.jac";'
+    replace = 'import {*} with "sample_bot/graph.jac";'
+
+    data_str = ""
+    find_str = ""
+    replace_str = ""
+
+    for line in data:  # concatenate it into one long string
+        data_str += line
+
+    for line in find:  # concatenate it into one long string
+        find_str += line
+
+    for line in replace:
+        replace_str += line
+
+    return data_str.replace(find, replace)  # return updated jac code
+
+
+def chat_setup():
+    try:
+        # base bot directory
+        bot_dir = curr_path / "sample_bot"
+        # Create temporary dir
+        base_path = tempfile.mkdtemp()
+
+        # Clone into temporary dir
+        git.Repo.clone_from(
```

Comment:
Do we have to download the git repository and get the templates there? The JacTalk binary already have access to all the jac files. Maybe it can just load it from file and then dump it out, instead of copying the files.